README.md

```diff
@@ -1,3 +1,16 @@
 # smallrepo
 
 foo
+bar
+baz
+xpto
```

Comment:
Single line suggestion (line6)

```suggestion
xptozzzzzzzzz
```

Reply:
Reply to https://github.com/anotherkassio/smallrepo/pull/2#discussion_r718627542

Reply:
Resolved!